packages/ag-charts-react/src/index.ts

```diff
@@ -26,7 +26,7 @@ interface BaseChartProps {
 function getOptions(options: AgChartOptions, containerRef: RefObject<HTMLElement | null>): AgChartOptions {
     return {
         ...options,
-        container: containerRef.current,
+        container: containerRef.current ?? undefined,
```

Comment:
Why was this null?

Reply:
We can probably revert this part tbh, I was trying to get rid of the support for `null` in the options contract, but it would mean changing ALL examples, so needs a bit more thought.

Reply:
Reverted